packages/backend/config/default.json

```diff
@@ -24,11 +24,21 @@
       "chunk": 100,
       "delay": 5000
     },
-    "sendgrid": {
-      "senderEmail": "no-reply@example",
-      "apiKey": "SG.example",
-      "confirmationTemplateId": "confirmationTemplateId",
-      "resetPasswordTemplateId": "resetPasswordTemplateId"
+    "mail": {
```

Comment:
Let's leave the old config available two for a couple of versions in order not to break backward compatibility.